katas/6kyu/Create phone number/test_create_phone_number.py

```diff
@@ -1,10 +1,18 @@
 import importlib
+import pytest
+
 from asserts.asserts import assert_true
+from asserts.testcase import Case
+
+solution = importlib.import_module('katas.6kyu.Create phone number.solution').create_phone_number
 
-create_phone_number = importlib.import_module('katas.6kyu.Create phone number.solution').create_phone_number
+cases = [
+    Case([1, 2, 3, 4, 5, 6, 7, 8, 9, 0], '(123) 456-7890'),
+    Case([1, 1, 1, 1, 1, 1, 1, 1, 1, 1], '(111) 111-1111'),
+]
 
 
 class TestSolution:
-    def test_create_phone_number(self):
-        assert_true(create_phone_number([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]), '(123) 456-7890')
-        assert_true(create_phone_number([1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), "(111) 111-1111")
+    @pytest.mark.parametrize('test', cases, ids=[f'{test.test_data}' for test in cases])
+    def test_consonant_value(self, test):
```

Comment:
it should be `test_create_phone_number `

Reply:
done